cryptoki-sys/src/lib.rs

```diff
@@ -97,699 +97,14 @@ pub type CK_EC_KDF_TYPE = CK_ULONG;
-pub const CKR_SESSION_PARALLEL_NOT_SUPPORTED: CK_RV = 0x000000B4;
-pub const CKR_SESSION_READ_ONLY: CK_RV = 0x000000B5;
-pub const CKR_SESSION_EXISTS: CK_RV = 0x000000B6;
-pub const CKR_SESSION_READ_ONLY_EXISTS: CK_RV = 0x000000B7;
-pub const CKR_SESSION_READ_WRITE_SO_EXISTS: CK_RV = 0x000000B8;
-pub const CKR_SIGNATURE_INVALID: CK_RV = 0x000000C0;
-pub const CKR_SIGNATURE_LEN_RANGE: CK_RV = 0x000000C1;
-pub const CKR_TEMPLATE_INCOMPLETE: CK_RV = 0x000000D0;
-pub const CKR_TEMPLATE_INCONSISTENT: CK_RV = 0x000000D1;
-pub const CKR_TOKEN_NOT_PRESENT: CK_RV = 0x000000E0;
-pub const CKR_TOKEN_NOT_RECOGNIZED: CK_RV = 0x000000E1;
-pub const CKR_TOKEN_WRITE_PROTECTED: CK_RV = 0x000000E2;
-pub const CKR_UNWRAPPING_KEY_HANDLE_INVALID: CK_RV = 0x000000F0;
-pub const CKR_UNWRAPPING_KEY_SIZE_RANGE: CK_RV = 0x000000F1;
-pub const CKR_UNWRAPPING_KEY_TYPE_INCONSISTENT: CK_RV = 0x000000F2;
-pub const CKR_USER_ALREADY_LOGGED_IN: CK_RV = 0x00000100;
-pub const CKR_USER_NOT_LOGGED_IN: CK_RV = 0x00000101;
-pub const CKR_USER_PIN_NOT_INITIALIZED: CK_RV = 0x00000102;
-pub const CKR_USER_TYPE_INVALID: CK_RV = 0x00000103;
-pub const CKR_USER_ANOTHER_ALREADY_LOGGED_IN: CK_RV = 0x00000104;
-pub const CKR_USER_TOO_MANY_TYPES: CK_RV = 0x00000105;
-pub const CKR_WRAPPED_KEY_INVALID: CK_RV = 0x00000110;
-pub const CKR_WRAPPED_KEY_LEN_RANGE: CK_RV = 0x00000112;
-pub const CKR_WRAPPING_KEY_HANDLE_INVALID: CK_RV = 0x00000113;
-pub const CKR_WRAPPING_KEY_SIZE_RANGE: CK_RV = 0x00000114;
-pub const CKR_WRAPPING_KEY_TYPE_INCONSISTENT: CK_RV = 0x00000115;
-pub const CKR_RANDOM_SEED_NOT_SUPPORTED: CK_RV = 0x00000120;
-pub const CKR_RANDOM_NO_RNG: CK_RV = 0x00000121;
-pub const CKR_DOMAIN_PARAMS_INVALID: CK_RV = 0x00000130;
-pub const CKR_BUFFER_TOO_SMALL: CK_RV = 0x00000150;
-pub const CKR_SAVED_STATE_INVALID: CK_RV = 0x00000160;
-pub const CKR_INFORMATION_SENSITIVE: CK_RV = 0x00000170;
-pub const CKR_STATE_UNSAVEABLE: CK_RV = 0x00000180;
-pub const CKR_CRYPTOKI_NOT_INITIALIZED: CK_RV = 0x00000190;
-pub const CKR_CRYPTOKI_ALREADY_INITIALIZED: CK_RV = 0x00000191;
-pub const CKR_MUTEX_BAD: CK_RV = 0x000001A0;
-pub const CKR_MUTEX_NOT_LOCKED: CK_RV = 0x000001A1;
-pub const CKR_NEW_PIN_MODE: CK_RV = 0x000001B0;
-pub const CKR_NEXT_OTP: CK_RV = 0x000001B1;
-pub const CKR_EXCEEDED_MAX_ITERATIONS: CK_RV = 0x000001B5;
-pub const CKR_FIPS_SELF_TEST_FAILED: CK_RV = 0x000001B6;
-pub const CKR_LIBRARY_LOAD_FAILED: CK_RV = 0x000001B7;
-pub const CKR_PIN_TOO_WEAK: CK_RV = 0x000001B8;
-pub const CKR_PUBLIC_KEY_INVALID: CK_RV = 0x000001B9;
-pub const CKR_FUNCTION_REJECTED: CK_RV = 0x00000200;
+pub const CKK_EC_MONTGOMERY: CK_KEY_TYPE = 0x00000041;
 pub const CKR_VENDOR_DEFINED: CK_RV = 0x80000000;
-pub const CKZ_DATA_SPECIFIED: CK_RSA_PKCS_OAEP_SOURCE_TYPE = 0x00000001;
-pub const CK_FALSE: CK_BBOOL = 0;
-pub const CK_TRUE: CK_BBOOL = 1;
+pub const CKR_CURVE_NOT_SUPPORTED: CK_RV = 0x00000140;
+pub const CKM_VENDOR_DEFINED: CK_MECHANISM_TYPE = 0x80000000;
```

Comment:
Same question as above..

Reply:
`#define CKM_VENDOR_DEFINED		((unsigned long) (1UL << 31))`
Latest version of bindgen still fails to generate bindings for these kind of definitions from the pkcs11.h.